templates/data/prow-periodics-data.yaml

```diff
@@ -29,7 +29,13 @@ templates:
                   image: gcr.io/k8s-prow/generic-autobumper:v20230226-ef2af11076
                   command: generic-autobumper
                   args:
-                    - --config=prow/autobump-config/eventing-tools-autobump-config.yaml
+                    - --config=ci/eventing-tools-autobump-config.yaml
+                  extra_refs:
+                    eventing-tools:
+                      - org: "kyma-project"
+                        repo: "eventing-tools"
+                        path_alias: "github.com/kyma-project/eventing-tools"
```

Comment:
Just a nit, `path_alias` it's not really needed for ref to be working. Especially here, there's no need to set up an alias, because this will be your working directory.

No need to remove it, but it's good if you keep that in mind next time!

Reply:
tried without `path_alias` but then templating fails and puts `<no value>` there
https://github.com/kyma-project/test-infra/pull/7215/commits/ee1f3eafcb0f7a18d0c4e133bff58a7073b34590

Reply:
🤣 Oh well, seems render templates doesn't support this use-case either...